constants.js

```diff
@@ -55,4 +55,7 @@ export const worldsHost = 'https://worlds.exokit.org';
 export const accountsHost = 'https://accounts.exokit.org';
 export const contractsHost = 'https://contracts.webaverse.com';
 export const presenceHost = 'worlds.webaverse.com';
-export const localstorageHost = 'https://localstorage.webaverse.com';
\ No newline at end of file
+export const localstorageHost = 'https://localstorage.webaverse.com';
+
+export const inventoryPageLimit = 6;
```

Comment:
I actually don't think these constants are very global. They are very tied to the design of the inventory, so should probably live in the inventory component  files.